sdk/go/pulumi/run.go

```diff
@@ -184,7 +184,7 @@ func getEnvInfo() RunInfo {
 		Stack:            os.Getenv(EnvStack),
 		Config:           config,
 		ConfigSecretKeys: configSecretKeys,
-		Parallel:         parallel,
+		Parallel:         int32(parallel), //nolint:gosec // guarded by strconv.ParseInt
```

Comment:
Should we cast it at the `ParseInt`?  Also what happens if that fails, we don't seem to have any error handling there :grimacing: It's probably not critical as I think if the syntax is wrong we'll get 0 here and it's going to be fine through the rest of the program.

Reply:
Why I wondered if ParseInt32 would be helpful